File: Code Python/Exercicio-4/Numero-inteiro-menor-que-1000.py
#Questão 8

num1=str(input("Digite um numero inteiro menor que 1000: \n"))
tam_num1 = len(num1)

if tam_num1 == 3:
    print("{} = {} centenas, {} dezenas e {} unidades".format(num1, num1[0], num1[1], num1[2]))

File: Code_C/Exercicio-04/Numero-inteiro-menor-que-1000.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

void pausar(){
	printf("\nPressione alguma tecla para continuar...");
	getch();
}

int main(){
	int num1, centena, dezena, unidade;
	int tam_num1;
	
	printf("Digite um numero inteiro menor que 1000: \n");
	scanf("%d", &num1);
	tam_num1 = (strlen, num1);
  
	centena  = (num1/100);
	dezena = (num1%100) / 10;
	unidade =  ((num1%100)%10);
	printf("%d = %d centenas, %d dezenas e %d unidades", num1, centena, dezena, unidade);
	
	pausar();
	return(0);
}
